packages/kbn-interpreter/src/public/browser_registries.js

```diff
@@ -18,7 +18,20 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import $script from 'scriptjs';
+
+function loadPath(path, callback) {
+  const script = document.createElement('script');
+
+  script.setAttribute('async', '');
+  script.setAttribute('nonce', window.__webpack_nonce__);
```

Comment:
I've generally seen nonce's set by the "backend" when rendering the HTML before serving it to the client. We don't really have the luxury in this situation, so am I correct in the theory that exposing the nonce on the window object allows scripts that are already able to run from essentially creating other scripts which can run. I believe we're in the same position without exposing the nonce on the window, because scripts could grab the `nonce` attribute value from existing scripts to do the same thing.

Reply:
Also, is it safe to assume we can't take advantage of `'strict-dynamic'` here because the `browser_registries.js` script isn't running with a hash/nonce already specified, so it can't implicitly inherit this?

Reply:
This is about protecting against data and user input, but not code we're serving ourselves. So yeah, this would allow any JavaScript that we've already deemed "safe" to load more JavaScript.

I think it's possible we could make `strict-dynamic` work, but I intentionally didn't see how far we could turn this knob for the sake of getting _some_ sort of reliable protection in place in time for 6.7.